docs/monorepo.md

```diff
@@ -1,15 +1,20 @@
 # Working with the Monorepo
 
-Calypso is a monorepo. In addition to the Calypso application, it also hosts a number of independent modules that are published to NPM.
+Calypso is a monorepo. In addition to the Calypso application, it also hosts a number of independent packages that are published to NPM.
```

Comment:
While we're updating these docs, let's add a little description of what the `apps` directory is about:

> ...to NPM and apps that are built and deployed as WordPress.com plugins.